packages/@aws-cdk/aws-efs/README.md

```diff
@@ -28,6 +28,20 @@ const fileSystem = new efs.FileSystem(this, 'MyEfsFileSystem', {
 });
 ```
 
+### Access Point
+
+An access point is an application-specific view into an EFS file system that applies an operating system user and group, and a file system path, to any file system request made through the access point. The operating system user and group override any identity information provided by the NFS client. The file system path is exposed as the access point's root directory. Applications using the access point can only access data in its own directory and below. To learn more, see [Mounting a File System Using EFS Access Points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html).
+
+Use `AccessPoint` to create the access point resources:
+
+```ts
+new AccessPoint(stack, 'MyAccessPoint', {
```

Comment:
How does one use an `AccessPoint` object in a client construct? By itself it doesn't magically do something.

I'd say this example is incomplete.

Also, can we add a `fileSystem.accessPoint({ ... })` convenience method?

Reply:
@rix0rrr  I got your point. I think `fileSystem.addAccessPoint({...})` makes sense. I'll add this into the PR today.

Reply:
@rix0rrr I probably still didn't get it.

Actually we will use `AccessPoint` in Lambda efs mount support https://github.com/aws/aws-cdk/pull/8602
I am planing to pass the `AccessPoint` to the Lambda filesystemConfigs like this:

```ts
 filesystemConfigs: {
    filesystem: lambda.LambdaFileSystem.fromEfsFileSystem(stack, accessPoint),
    localMountPath: '/mnt/msg',
  }
```

I also tried to implement `addAccessPoint` for `FileSystem` construct but this will create two `Resource`.